_bibliography/pint.bib

```diff
@@ -990,7 +1081,8 @@ @inbook{Ulbrich2007
 	pages = {145-168},
 	doi = {10.1137/1.9780898718935.ch7},
 	url = {https://dx.doi.org/10.1137/1.9780898718935.ch7},
-	year = {2007}
+	year = {2007},
+	publisher = {SIAM}
```

Comment:
article entries do not have the publisher, again to be . consistent with other entries.

Reply:
this was not consistent wrt to existing entries. I went through and made it consistent.